libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -39,12 +39,50 @@ import {
   NumberString,
 } from './ui_strings';
 
-const Ballot = styled.div`
+export type BmdBallotPrintType = 'vxMark' | 'vsap';
+
+/**
+ * Max margin required to keep the ballot header visible when the page is
+ * partially covered by the VSAP infeed hood.
+ */
+export const MAX_VSAP_TOP_MARGIN = '1.75in';
+
+/**
+ * Min margin required to keep the first row of contests visible when the page
+ * is partially covered by the VSAP infeed hood.
+ */
+export const MIN_VSAP_TOP_MARGIN = '0.5625in';
+
+interface Layout {
+  hideParties: boolean;
+  maxRows: number;
+  minContests: number;
+  topMargin?: typeof MAX_VSAP_TOP_MARGIN | typeof MIN_VSAP_TOP_MARGIN;
+}
```

Comment:
It's nice how these layout specs are defined as data. Very cool

I'm a little confused still by what `maxRows` means

Reply:
Yeah, good point - makes me think all of these fields could use documentation for clarity. Will update